x-pack/plugins/ml/public/application/datavisualizer/file_based/components/file_datavisualizer_view/file_datavisualizer_view.js

```diff
@@ -177,12 +177,22 @@ export class FileDataVisualizerView extends Component {
       });
     } catch (error) {
       console.error(error);
+
+      let serverErrorMsg;
+      const isErrorResponse =
```

Comment:
I see the function `isErrorResponse` above which does the same check

Reply:
Done in https://github.com/elastic/kibana/pull/60249/commits/d8926e25a3a4e1c7e3dd20a6342c15cba035bfc5